.gitignore

```diff
@@ -18,3 +18,6 @@ packages/databricks-sdk-js/src/config/testdata/azure/Library
 *.d.ts.map
 *.js.map
 tsconfig.tsbuildinfo
+
+# Telemetry file, automatically generated by packages/databricks-vscode/scripts/generateTelemetry.ts
+packages/databricks-vscode/telemetry.json
```

Comment:
This line should go into the `.gitignore` inside of `packages/databricks-vscode`